app/workers/articles/bust_multiple_caches_worker.rb

```diff
@@ -0,0 +1,15 @@
+module Articles
+  class BustMultipleCachesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(article_ids, cache_buster = "CacheBuster")
```

Comment:
Seeing as we never use this second argument, we always go with the default I think we can remove it and skip constantizing it below.

Reply:
Just for lore: the reason why the cache buster has stuck there it's because it used to be an instance object, not a class, so it was easier to test if passed as an argument. We can remove it everywhere now that it is :)

Reply:
ok guys, I'll remove it :)